<launch>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find frobo_nav)/config/frobo/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find frobo_nav)/config/frobo/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find frobo_nav)/config/frobo/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find frobo_nav)/config/frobo/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find frobo_nav)/config/frobo/base_local_planner_params.yaml" command="load" />
  </node>
  
</launch>
